test/get_previous_releases.py

```diff
@@ -258,6 +284,8 @@ def check_host(args) -> int:
             'x86_64-*-linux*': 'x86_64-linux-gnu',
             'x86_64-apple-darwin*': 'x86_64-apple-darwin',
             'aarch64-apple-darwin*': 'arm64-apple-darwin',
+            'x86_64-w64-mingw32': 'win64',
+            'x86_64-*-win*': 'win64',
```

Comment:
What is the purpose of this host triple? We don't make Windows binaries with any other host triple.